### PR TITLE
Adds webinar video to solutions/device-management/

### LINKED ISSUE #1023: Add device management webinar to device management solution page
### Description

We should add the device management webinar recording to our device management solution page. 

https://flowforge.com/solutions/device-management/
https://youtu.be/dvTgCBXCa5A

### Epic/Story

_No response_

### Have you provided an initial effort estimate for this issue?

I have provided an initial effort estimate

### PR DESCRIPTION
**Pull request by Yndira-FlowForge**
## Description

Replaced the UI image with the webinar video

## Related Issue(s)

Closes #1023

## Checklist

 - [x] I have read the [contribution guidelines](https://github.com/flowforge/flowforge/blob/main/CONTRIBUTING.md)
 - [x] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

### CODE REVIEW

**Comment by Yndira-FlowForge** [comment]
Hi @joepavitt I could use your thoughts on this one.

I've added a bg image for the video, I see it like this locally:

Screenshot 2023-08-11 at 14 56 33


But it isn't showing in the netlify preview:
Screenshot 2023-08-11 at 14 57 56

Will the image be displayed correctly if we merge this?

**Comment by joepavitt** [comment]
hmmm, good question, and not one I know the answer to straight away without digging into.

**Comment by joepavitt** [comment]
if you inspect the element in the Browser on the preview, you'll see that this image doesn't exist

Screenshot 2023-08-11 at 15 23 01

**Comment by joepavitt** [comment]
not sure why it's working locally, because your path is wrong, it needs an additional `../` at the start (your page is nested inside `/solutions` and you're needing to go up two levels to get to `./images`

**Comment by Yndira-FlowForge** [comment]
Thanks @joepavitt!!

File: src/solutions/device-management.njk [hunk 1]
```diff
@@ -77,15 +77,19 @@ description:
             </div>
         </div>
     </div>
+    <div class="w-full max-w-screen-lg m-auto pb-32 px-6">
+        
+            <div class="flex justify-center items-center md:w-full">
+                <lite-youtube videoid="dvTgCBXCa5A" class="aspect-video bg-cover bg-top bg-no-repeat border-2 border-black rounded-md drop-shadow-xl" style="background-image: url('../images/solutions/webinar-july-2023_video_tile.jpg'); width: 1080px; background-repeat: no-repeat;"></lite-youtube>
```
Comment (Yndira-FlowForge) [inline src/solutions/device-management.njk:83]:
```suggestion
                <lite-youtube videoid="dvTgCBXCa5A" class="aspect-video bg-cover bg-top bg-no-repeat border-2 border-black rounded-md drop-shadow-xl" style="background-image: url('../../images/solutions/webinar-july-2023_video_tile.jpg'); width: 1080px; background-repeat: no-repeat;"></lite-youtube>
```

**Review by iskerrett — APPROVED** [review]

**Pull request merged**